db/migrate/20231010211432_add_field_to_user.rb

```diff
@@ -0,0 +1,10 @@
+class AddFieldToUser < ActiveRecord::Migration[6.1]
```

Comment:
Please rename migration name to add_fields_to_user, not field as we are adding multiple fields in user table